pkg/model/tests/data/bootstrapscript_0.txt

```diff
@@ -132,12 +120,8 @@ function download-release() {
     ;;
   esac
 
-  # In case of failure checking integrity of release, retry.
   cd ${INSTALL_DIR}/bin
-  until try-download-release; do
-    sleep 15
-    echo "Couldn't download release. Retrying..."
-  done
+  try-download-release
```

Comment:
`try-download-release` is doing very little now. Maybe just drop it and move what's left to `download-or-bust`.